lua/entities/gmod_wire_hsranger/init.lua

```diff
@@ -18,7 +18,9 @@ local MODEL = Model( "models/jaanus/wiretool/wiretool_range.mdl" )
 	5 = SkewY
 	6 = Trace Water
 	
-	7 = unused

```

Comment:
Low level hardware protocols often seem to have unused reserved spaces for future development, like TCP headers. Despite being totally unnecessary in Lua, I think they're here for flavour, and to nicely round out the half byte of 0-7.